compiler/rustc_middle/src/ty/inhabitedness/mod.rs

```diff
@@ -56,7 +56,8 @@ impl<'tcx> TyCtxt<'tcx> {
     /// Checks whether a type is visibly uninhabited from a particular module.
     ///
     /// # Example
-    /// ```rust
+    /// ```ignore HELP (should I just add 'use super::Void;' to 'mod c' or does that defeat the point of the example?)
+    /// #![feature(never_type)]
```

Comment:
"visibly" here is referring to exhaustiveness, not name resolution. `use super::Void` should be fine.